src/app/frontend/chrome/nav/style.scss

```diff
@@ -35,7 +35,8 @@
     }
 
     .kd-nav-items {
-      padding: $content-padding 3px $content-padding $content-padding;
+      padding: $content-padding;
+      padding-inline-end: 3px;
```

Comment:
Can you explain this change?

Reply:
Like [this](https://github.com/kubernetes/dashboard/pull/5701#discussion_r535066281) one, I changed it to change the padding side automatically based on language.